js/collaborator-agent/src/agent/instructions.ts

```diff
@@ -0,0 +1,299 @@
+// Prompt instructions for different capabilities of the Collaborator bot
+
+// Get current date with day of week and reuse it
+const now = new Date();
```

Comment:
Do this in a function call. Once your server spins up, this date will not change. So if your server's running for 2 days, this will be out of date.